static/src/javascripts/projects/common/modules/commercial/third-party-tags/outbrain.js

```diff
@@ -101,11 +101,15 @@ define([
                 || _.contains(['politics', 'world', 'business', 'commentisfree'], config.page.section.toLowerCase()) ? 'sections' : 'all';
         },
 
+        identityPolicy: function () {
```

Comment:
Perhaps we should now start adding this logic to `commercial-feature-policies.js`. It might make your testing simpler if you could just verify that `outbrain.js` just listens to the boolean policy flag, and that the policy itself returns the correct boolean, I think.

We don't have to do this now, but we should probably have a plan.